dronecore/backend.py

```diff
@@ -0,0 +1,90 @@
+""" Backend """
```

Comment:
Wording question: isn't this what we call `Core`? What we call the "backend" is really just the gRPC server side (C++). So the `Core` spins up a backend.

Reply:
Okay, Let's rename that to Core! :-)